### DISCUSSION TITLE
AI倫理研究者Timnit Gebru因論文爭議被迫離開Google，員工與學術界串連數千人討公道

### DISCUSSION THREAD

**Discussion by ruthchen1020** [1101-GS4719-C/Python-Community-GS4719-C#888] [General]
Google知名AI倫理學家Timnit Gebru表示遭公司報復性解僱，她來自衣索比亞，是一位電腦科學家，專業領域是研究人工智慧的公平性，而在Google，Timnit Gebru是人工智慧倫理團隊的共同領導人。這個團隊致力於避免模型在特定情況出現偏差，解決像是人臉辨識系統，在處理黑人女性臉孔錯誤率偏高等問題。
https://www.ithome.com.tw/news/141504

**Comment by ching020721** [comment]
這個議題在我去年修人工智慧倫理的時候也有探討過，我認為google不應該報復性解雇這名科學家，這樣只會引起很多人的不滿。

**Comment by manipulateed** [comment]
雖然她被解僱，但可以從她發表的文章內容發現，人工智慧在開發的過程中會造成能源與倫理層面的問題，我們應該在訓練時公開給大眾去審視，才能好好正視這些問題。

**Comment by CPCpyt** [comment]
我覺得這對於他來說不公平，報復性解雇在職場的任何地方都不該出現，
希望人資可以更注重這個部分和社會觀感。
